### PR TITLE
Added v2plugin install support

### PR DESCRIPTION
**Pull request by gkvijay**
https://github.com/contiv/ansible/pull/344 needs to merged before this

### CODE REVIEW

File: install/ansible/uninstall_swarm.sh [hunk 1]
```diff
@@ -137,10 +142,10 @@ else
 	ans_opts=$(printf '%q', $ans_opts)" --private-key $def_ans_key -u $ans_user"
 fi
 echo "Starting the uninstaller container"
-image_name="contiv/install:__CONTIV_INSTALL_VERSION__"
+image_name="contiv/install:1.0.0"
```
Comment (rhim) [inline install/ansible/uninstall_swarm.sh]: Why is this hard coded to 1.0.0

Comment (gkvijay) [inline install/ansible/uninstall_swarm.sh]: Missed it. Will revert

File: install/ansible/uninstall.sh [hunk 2]
```diff
@@ -140,8 +144,13 @@ fi
 echo "Uninstalling Contiv"
 
 # Uninstall contiv & API Proxy
-echo '- include: uninstall_auth_proxy.yml' >$ansible_path/uninstall_plays.yml
-echo '- include: uninstall_contiv.yml' >>$ansible_path/uninstall_plays.yml
+if [ $uninstall_v2plugin == true ]; then
+	echo '- include: uninstall_auth_proxy.yml' >$ansible_path/uninstall_plays.yml
+	echo '- include: uninstall_v2plugin.yml' >$ansible_path/uninstall_plays.yml
```
Comment (dseevr) [inline install/ansible/uninstall.sh]: This line overwrites uninstall_plays.yml, but the equivalent line in the `else` branch appends it.  Is this correct?

Comment (gkvijay) [inline install/ansible/uninstall.sh]: should be append. changed it.

**Pull request merged**